docs/source/release/v4.2.0/framework.rst

```diff
@@ -15,6 +15,7 @@ Concepts
 Algorithms
 ----------
 * ref:`MaskAngle <algm-MaskAngle>` has an additional option of ``Angle='InPlane'``
```

Comment:
We usually discourage changes that are not related but the `ref:` part here is wrong. Any chance you could update it to `:ref:` to fix the link?